packages/core/package.json

```diff
@@ -11,6 +11,7 @@
   "dependencies": {
     "@cmsgov/design-system-support": "^1.3.0",
     "classnames": "^2.2.5",
+    "core-js": "^2.5.1",
```

Comment:
@1Copenut FYI. This needed added to the core package's `package.json` since each design system package can be installed separately, so any dependency imported into a core component need added.